Kafka/README.md

```diff
@@ -86,6 +86,7 @@ A full skeleton MDB is shown below
     @ActivationConfigProperty(propertyName = "clientId", propertyValue = "testClient"),
     @ActivationConfigProperty(propertyName = "groupIdConfig", propertyValue = "testGroup"),
     @ActivationConfigProperty(propertyName = "topics", propertyValue = "test,test2"),
+    //@ActivationConfigProperty(propertyName = "regexTopics", propertyValue = "test.*"),
```

Comment:
Maybe add doc about this property above and clarify the regex syntax by pointing to the Pattern classes Javadoc?

Reply:
Good idea - but could not the existing topic property documented anywhere